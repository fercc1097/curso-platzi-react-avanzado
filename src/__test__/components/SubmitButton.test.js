import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';

import { SubmitButton } from '../../components/SubmitButton/index';

describe('<SubmitButton />', () => {
  test('Render del componente Footer ', () => {
    const submitButton = mount(<SubmitButton />);
    expect(submitButton.length).toEqual(1);
  });
});

describe('FavButton Snapshot', () => {
  test('Comprobar la Ui del componente Footer', () => {
    const submitButton = create(<SubmitButton />).toJSON();
    expect(submitButton).toMatchSnapshot();
  });
});
