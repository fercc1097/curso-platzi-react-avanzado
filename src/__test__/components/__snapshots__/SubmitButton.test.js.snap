// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavButton Snapshot Comprobar la Ui del componente Footer 1`] = `
<button
  className="sc-bdnylx LQFXu"
/>
`;
